tests/Microsoft.Identity.Test.Common/MsalTestConstants.cs

```diff
@@ -141,4 +141,15 @@ public static string CreateUserIdentifier(string uid, string utid)
         public static readonly ClientCredential CredentialWithSecret = new ClientCredential(ClientSecret);
 #endif
     }
+
+    internal static class Adfs2019LabConstants
+    {
+        public const string Authority = "https://fs.msidlab8.com/adfs";
+        public const string AppId = "TestAppIdentifier";
+        public const string PublicClientId = "PublicClientId";
+        public const string ConfidentialClientId = "ConfidentialClientId";
+        public const string ClientSecret = "olCpEkQu8WvmTln1D4cDO1-0DyJWKV5m6gvSBDcM";

```

Comment:
>public const string ClientSecret = "olCpEkQu8WvmTln1D4cDO1-0DyJWKV5m6gvSBDcM"; [](start = 8, length = 78)

please role the secret and use the cert instead. the build machine has access to KeyVault where it should get the secret from. @trwalke

Reply:
How will cert rollover occur on the Adfs machine? Someone will need to handle this before the cert expires each time

Reply:
Oh I see, @reed1995 I think what @henrik-me is saying here is we should store your secret in our keyvault and acquire it at runtime from the keyvault via a cert installed on the machine. We already have tests that do this. Ill follow up offline

Reply:
Also your identity will allow you to run this test locally as it will authenticate as you to get the secret

Reply:
nvm...Travis helped me out with this